dask_expr/_expr.py

```diff
@@ -2603,6 +2603,51 @@ def _fusion_pass(expr):
     return expr
 
 
+class Shift(MapOverlap):
+    _parameters = ["frame", "periods", "freq"]
+    _defaults = {"periods": 1, "freq": None}
+
+    func = M.shift
+    enforce_metadata = True
+    before = 0
+    after = 0
+
+    def _divisions(self):
+        return self.frame.divisions
```

Comment:
I think divisions change in shift? Not totally sure

Reply:
you can do `ddf.shift(periods=2).enforce_runtime_divisions()` to see if the divisions are working as expected